XRTK-Core/Packages/com.xrtk.core/Definitions/InputSystem/MixedRealityInputAction.cs

```diff
@@ -30,32 +48,61 @@ public MixedRealityInputAction(uint id, string description, AxisType axisConstra
         /// <summary>
         /// Default input action that doesn't represent any defined action.
         /// </summary>
-        public static MixedRealityInputAction None { get; } = new MixedRealityInputAction(0, "None");
+        public static MixedRealityInputAction None { get; } = new MixedRealityInputAction(default, 0, "None");
```

Comment:
```suggestion
        public static readonly MixedRealityInputAction None = new MixedRealityInputAction(default, 0, "None");
```

Reply:
This will not compile until Unity supports c# 8

Reply:
Oh, I see, this is now a field.